src/utils.js

```diff
@@ -417,7 +417,30 @@ export function computeLinesDefault({ctx, text, max, wordSpacing, letterSpacing}
     }
 
     if ([...line].length !== 0) {
-        lines.push(line);
+        if (options.wordwrap) {
+            line
```

Comment:
The wordwrap option should be considered above before the breakpoint switch...case. You also should only check the last part and not the entire line. (Think about a line with 1000 characters and only the last 10char word is to long)

Reply:
Right, I moved it just before the process loop over text parts and compute the lines, and add a breakpoint of type `SHY` for each new line.